internal/handlers/commonparams/extract_params.go

```diff
@@ -88,7 +88,8 @@ func ExtractParams(doc *types.Document, command string, value any, l *zap.Logger
 		lookup := key
 
 		// If the key is the same as the command name, then it is a collection name.
-		if key == command {
+		// Depending on the driver, the key may be camel case or lower case for a collection name.
+		if strings.EqualFold(key, command) {
```

Comment:
This is regression from `extract_params.go`, currently `findAndModify` command doesn't work on js driver used by `task testjs`. I added test to cover 

https://github.com/FerretDB/FerretDB/blob/ac2410875a13f97017e9cded29b816b5177d463d/integration/findandmodify_test.go#L129-L132